main.py

```diff
@@ -1,10 +1,44 @@
+from json import load
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players_json:
+        players_data = load(players_json)
+
+    for nickname in players_data:
+        race, _ = Race.objects.get_or_create(
+            name=players_data[nickname]["race"]["name"],
+            description=players_data[nickname]["race"]["description"]
+        )
+
+        for skill_data in players_data[nickname]["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race
+            )
+
+        guild_data = players_data[nickname].get("guild")
+
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=players_data[nickname]["guild"]["name"],
+                description=players_data[nickname]["guild"]["description"],
+            )
+        else:
+            guild = None
+
+        Player.objects.get_or_create(
+            nickname=nickname,
+            email=players_data[nickname]["email"],
```

Comment:
Ensure that the email addresses in the JSON data are unique if you intend for each player to have a distinct email. The current model does not enforce email uniqueness, which might lead to duplicate entries if the same email is used for multiple players.